deploy/gatekeeper.yaml

```diff
@@ -101,6 +101,12 @@ status:
   conditions: []
   storedVersions: []
 ---
+apiVersion: v1
```

Comment:
I think we also need to explicitly bind the serviceaccount to the Pod:

https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#use-the-default-service-account-to-access-the-api-server